examples/tests/sdk_tests/queue/dead-letter-queue.test.w

```diff
@@ -0,0 +1,56 @@
+bring cloud;
+bring util;
+
+let c = new cloud.Counter();
+
+let key_without_retries = "without_retries";
+let dlq_without_retries = new cloud.Queue() as "dlq without retries";
+let queue_without_retries = new cloud.Queue(
+  dlq: { queue: dlq_without_retries }
+) as "queue without retries";
+queue_without_retries.setConsumer(inflight (msg: str) => {
+  if msg == "fail" {
+    c.inc(1, key_without_retries);
+    throw "error";
+  }
+});
+
+new std.Test(inflight () => {
+  queue_without_retries.push("Hello");
+  queue_without_retries.push("fail");
+  queue_without_retries.push("World!");
+
+  // wait until it executes once.
+  assert(util.waitUntil(inflight () => { return c.peek(key_without_retries) == 1; }));
+
+  // check if the "fail" message has arrived at the dead-letter queue
+  assert(util.waitUntil(inflight () => { return dlq_without_retries.pop() == "fail"; }));
+}, timeout: 5m) as "one execution and send fail message to dead-letter queue";
```

Comment:
In the simulator, it runs quite fast, but on AWS, it takes longer for the reason you mentioned. I set this value so that GitHub can run the test on AWS.